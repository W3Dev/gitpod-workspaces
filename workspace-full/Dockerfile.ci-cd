FROM gitpod/workspace-full
LABEL org.opencontainers.image.source = "https://github.com/W3Dev/gitpod-workspaces"

RUN sudo apt-get update -y

# Install Gitlab Runner
RUN curl -L "https://packages.gitlab.com/install/repositories/runner/gitlab-runner/script.deb.sh" | sudo bash
RUN sudo apt-get install gitlab-runner -y

# Install act for running Github Actions locally
RUN brew install act