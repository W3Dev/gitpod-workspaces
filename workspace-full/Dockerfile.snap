FROM gitpod/workspace-full-vnc
LABEL org.opencontainers.image.source = "https://github.com/W3Dev/gitpod-workspaces"

ENV container docker
ENV PATH /snap/bin:$PATH
ADD snap /usr/local/bin/snap
RUN sudo apt-get update
RUN sudo apt-get install -y snapd squashfuse fuse
RUN sudo systemctl enable snapd
STOPSIGNAL SIGRTMIN+3

# Not Working